eq-author/src/App/page/Design/QuestionPageEditor/MetaEditor.test.js

```diff
@@ -16,6 +18,7 @@ describe("MetaEditor", () => {
       onUpdate: jest.fn(),
       fetchAnswers: jest.fn(),
       onChangeUpdate: jest.fn(),
+      getValidationError: getValidationError,
```

Comment:
Can we define it in beforeEach as well so that we don't have the call information shared between tests

Reply:
We also have no tests using function. Are we missing it?

Reply:
Changed to jest.fn() as no tests using the function.